package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "product-availability",
+  "private": true,
+  "license": "UNLICENSED",
+  "scripts": {
+    "lint": "eslint --ext js,jsx,ts,tsx .",
+    "test": "cd ./react && yarn test",
+    "format": "prettier --write \"**/*.{ts,tsx,js,jsx,json}\""
+  },
+  "husky": {
```

Comment:
This husky setting doesn't need anymore, you should run this command `npx husky add .husky/pre-commit "yarn lint-staged"`. Take a look here https://typicode.github.io/husky/#/?id=add-a-hook